modules/modules.nix

```
@@ -116,6 +116,7 @@ let
     (loadModule ./services/stalonetray.nix { })
     (loadModule ./services/status-notifier-watcher.nix { })
     (loadModule ./services/syncthing.nix { })
+    (loadModule ./services/rsibreak.nix { })
```

Since this module uses systemd it should be limited to Linux:
```suggestion
    (loadModule ./services/rsibreak.nix { condition = hostPlatform.isLinux; })
```